profiles/base.yaml

```diff
@@ -1120,6 +1120,7 @@ held_athletics_items:
 # https://elanthipedia.play.net/Lich_script_repository#astrology
 astrology_buffs:
 have_telescope: false
+telescope_storage:
```

Comment:
Do I need this if I also have it in base-empty, I can't remember.

Reply:
Yes